person.rb

```diff
@@ -4,14 +4,15 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(age, name = 'unknow', parent_permission: true)
     @id = Random.rand(1..100)
     @name = name
     @age = age
     @parent_permission = parent_permission
     super()
+    @rentals = []
   end
```

Comment:
- [ ] Kinldy makes sure a person has `many` rentals as specified in the project requirements.

Hint: You may consider saying something like :
```rb
def add_rental(book, date)
    # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
  end
```